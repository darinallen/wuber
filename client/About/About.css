.about {
  color: #7788BB;
}