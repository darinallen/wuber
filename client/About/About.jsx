import React from 'react';
import { Link } from 'react-router-dom';
import './About.css';

const About = () => (
  <div className="about">
    <h1>You are on the About page!</h1>
    <Link to="/">Home</Link>
  </div>
);

export default About;
