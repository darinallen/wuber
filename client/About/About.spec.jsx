import React from 'react';
import { render } from 'enzyme';
import { MemoryRouter } from 'react-router'; // Needed to preserve the history of your “URL” in memory
import About from '../About/About';

test('About component renders correctly', () => {
  const component = render(<MemoryRouter><About /></MemoryRouter>);
  expect(component).toMatchSnapshot();
});
