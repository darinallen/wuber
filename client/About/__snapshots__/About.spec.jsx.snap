// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component renders correctly 1`] = `
<div
  class="about"
>
  <h1>
    You are on the About page!
  </h1>
  <a
    href="/"
  >
    Home
  </a>
</div>
`;
